### PR TITLE
Bump SDK to v1.0.1

### PR DESCRIPTION
**Pull request by dmanjunath**
### Description
Upgrade SDK to v1.0.1

### Dragons
Nothing of note in client. Full Changelog for sdk is here https://github.com/AudiusProject/audius-protocol/commit/ff85889a8b534b90af5fdd0a777f6ac35083b49f
*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?
Pointed local client to stage and ran create user, track upload, play track

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?
Sentry and amplitude error rates
*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###
No feature flags
*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by sliptype — APPROVED** [review]
Could you also bump the one in `packages/common/package.json`?

For future ref, you should be able to do `npx lerna add @audius/sdk@1.0.1` from the root

File: packages/stems/package.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
     "dist"
   ],
   "dependencies": {
+    "@audius/sdk": "1.0.1",
```
Comment (dmanjunath) [inline packages/stems/package.json:100]: @sliptype wasn't sure if this was supposed to happen

**Comment by dmanjunath** [comment]
@sliptype i ran that command and pushed the result of that. i don't think stems should get this sdk bump right?

**Comment by sliptype** [comment]
> @sliptype i ran that command and pushed the result of that. i don't think stems should get this sdk bump right?

It should be fine to bump the one in stems too! What's with the commen password check change though?

**Comment by dmanjunath** [comment]
> > @sliptype i ran that command and pushed the result of that. i don't think stems should get this sdk bump right?
> 
> It should be fine to bump the one in stems too! What's with the commen password check change though?

whoops that was a change i used during local testing because i was getting the digest error so i forced it to pass validation. definitely going to be removing that and will test again 🙂

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dm-upgrade-sdk-1.0.1

**Comment by dmanjunath** [comment]
okay @sliptype final version if you want to give it a once over

**Review by sliptype — APPROVED** [review]
Looks good!

**Pull request merged**